### ISSUE TITLE
White street names and arrows

### ISSUE THREAD

**Issue by crashbangwallop** [barryhunter/classygmaps#86]
I’d like to echo fasterfaster’s suggestion (in another post) of a graphics-free screenshot capability. To be more specific though, I could go further and suggest a ‘view’ without the street names and white direction arrows. I’m a painter, and previously used the Classic streetview for sourcing and combining landscapes and skies. Is this possible? 

I found your project last night while looking through the google maps help forum – such a relief that I can resume that aspect of my artwork without cumbersome workarounds. Wonderful to see the split screen again – for me it’s so useful to be able to record a location of a particular feature with a simple screenshot. I’ll be passing on the news to others this morning.

Good luck with the development of ClassyGMaps - Cheers

**Comment by barryhunter** [comment]
Dont think can do hide this stiff, the API exposes no controls for it. 

Its possibly somewhat deliberate, to make it less attractive to try to steal their imagery. Any screenshots retain the Google look.

**Comment by crashbangwallop** [comment]
Fair enough Barry, they are relatively small flies in the general good news ointment. 

Thanks for doing this, it's so much easier to work with than the present streetview. 

Yours

Keith
  ----- Original Message -----

**Comment by fasterfaster** [comment]
The workaround I've been using to temporarily hide on-screen elements, so I can do clean screenshots, requires the Seamonkey/Firefox add-ons:

* Adblock Plus - http://www.adblockplus.org/en/
* Adblock Plus Element Hiding Helper - https://adblockplus.org/elemhidehelper

Adblock plus an icon in the status bar.  Click on it, select "select an element to hide", then select the elements on the page to hide.  Note the yellow box in the upper left corner that gives you hotkeys for the element hiding functions.  You can hide it with "H" if its in the way.  Use "W" to widen your selection, and "N" to narrow it down.

When you have the exact element you want selected, click on it and a box with options will pop up.  Tick the "preview" box in the upper right corner, but LEAVE THE BOX OPEN.  Do this for each element you want to hide.

Now you can go back to ClassyGMaps underneath, and all of your elements will be temporarily gone.  After you're done doing your screenshots, click "Cancel" on each of those option boxes, and the elements will be back.

While you *could* have AdBlock hide them permanently, they're options you'll need to use ClassyGmaps, so it's not suggestible to do that.  But you will have to repeat all of this anytime you want to go back to the site and do screenshots.  That's why I was hoping a "screenshot mode" could be created to make all of that a lot easier :-)

**Comment by crashbangwallop** [comment]
Wow! Thanks fasterfaster! I'll give that a try when I have more time over the weekend. I was just going to fiddle around on my source screenshots with photoshop, but this looks like that'll save me doing that. Thanks :)

**Comment by WildRasberry** [comment]
There is another option that's quick and easy for regular Street View (not ClassyGMaps) but unfortunately leaves street names... it's for Firefox and it's called CleanMap. You end up with a button to clear everything - except street names - and turn it back on again. You can get it here:
http://dddavemaps.blogspot.ca/2015/05/cleanmap-bookmarklet.html

**Comment by crashbangwallop** [comment]
Thank you WIldRaspberry, that's very useful (works well on Chrome too) and simple to use. I'll pass that on :)
